toolchain/gnu/Makefile

```diff
@@ -0,0 +1,232 @@
+TOPDIR = $(CURDIR)
+
+include $(TOPDIR)/common.mk
+
+all: package
+
+build-host: gmp-install mpfr-install mpc-install isl-install cloog-install
+
+clean-host: gmp-clean mpfr-clean mpc-clean isl-clean cloog-clean
+	$(RM) -r $(HOSTDIR) $(HOSTBUILD)
+
+build-%-mimiker-elf: build-host binutils-unpack gcc-unpack gdb-unpack
+	$(MAKE) -C $*-mimiker-elf
+
+%-mimiker-elf_$(VERSION)_amd64.deb: build-%-mimiker-elf
+	$(MAKE) -C $*-mimiker-elf package
+
+clean-%-mimiker-elf:
+	$(MAKE) -C $*-mimiker-elf clean
+	$(RM) $*-.deb
+
+build: $(TARGETS:%=build-%-mimiker-elf)
+
+package: $(TARGETS:%=%-mimiker-elf_$(VERSION)_amd64.deb)
+
+clean: $(TARGETS:%=clean-%-mimiker-elf) clean-host
+	$(RM) -r sources
+
+.PHONY: all build clean build-host clean-host
+
+### GNU Binutils
+
+$(SRCDIR)/$(BINUTILS).tar.xz:
+	$(CURL) -o $@ $(BINUTILS-URL)
+
+binutils-unpack: $(SRCDIR)/binutils/.unpack
+$(SRCDIR)/binutils/.unpack: $(SRCDIR)/$(BINUTILS).tar.xz
```

Comment:
Why do we need the `.unpack` file? Can't we use `$(SRCDIR)/binutils` instead (with no `touch $@`)?

Reply:
Without that `.unpack` file Make rebuilds the targets twice. I'm not sure why it happens, but modify time of the directory is constantly changed. So in the result, Make re-triggers the target. 

I don't think that these `.unpack` files is a huge problem.